test/IntegrationTests/BuildTests.DistributionStructure_alpine-linux-arm64.verified.txt

```diff
@@ -1,15 +1,13 @@
 ﻿[
-  /AdditionalDeps/shared/Microsoft.NETCore.App/6.0.0/OpenTelemetry.AutoInstrumentation.AdditionalDeps.deps.json,
-  /AdditionalDeps/shared/Microsoft.NETCore.App/7.0.0/OpenTelemetry.AutoInstrumentation.AdditionalDeps.deps.json,
   /AdditionalDeps/shared/Microsoft.NETCore.App/8.0.0/OpenTelemetry.AutoInstrumentation.AdditionalDeps.deps.json,
+  /AdditionalDeps/shared/Microsoft.NETCore.App/9.0.0/OpenTelemetry.AutoInstrumentation.AdditionalDeps.deps.json,
   /LICENSE,
   /instrument.sh,
   /linux-musl-arm64/OpenTelemetry.AutoInstrumentation.Native.so,
   /net/Google.Protobuf.dll,
   /net/Grpc.Core.Api.dll,
   /net/Grpc.Net.Client.dll,
   /net/Grpc.Net.Common.dll,
-  /net/Microsoft.Extensions.Diagnostics.Abstractions.dll,
```

Comment:
I guess this is now shipped by the framework? 

Could prob remove it here also then.
https://github.com/Kielek/opentelemetry-dotnet-instrumentation/blob/d992b5fd8518ceb7ae1aa9eaeb2fee2882caae88/LICENSE#L271

Reply:
It is shipped for .NET Fx -> should stay in Liecense file.

It is referenced in 8.0.0.0 version. I is included in .NET8 -> it is always available at least in this version. No need to shipping it.